src/angularComponent/ngcOmnibox.html

```diff
@@ -1 +1,5 @@
-<h1>Hello world!</h1>
+<input type="text"
+    ng-model="omnibox.ngModel"
+    ng-attr-placeholder="{{::omnibox.placeholder || undefined}}"
```

Comment:
@wei No, for `ng-attr` `undefined` is the only way to get it to _remove_ the attribute as opposed to just removing its value.

sgtm about bind-once